gen/gen.go

```diff
@@ -90,7 +90,7 @@ func outputSource(status string, fileName string, src []byte) error {
 }
 
 // Gen generates the exercise cases_test.go file from the relevant canonical-data.json
-func Gen(exercise string, j interface{}, t *template.Template) error {
+func Gen(exercise string, j interface{}, t *template.Template, tmp func()) error {
```

Comment:
ah right, all other calls to `Gen` are going to need to change here, unless the `tmp` becomes unnecessary. And `tmp` is going to need a better name (`postProcess` or something like that)